db/config.cc

```diff
@@ -968,6 +968,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_repair_based_node_ops(this, "enable_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, true, "Set true to use enable repair based node operations instead of streaming based.")
     , allowed_repair_based_node_ops(this, "allowed_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, "replace,removenode,rebuild,bootstrap,decommission", "A comma separated list of node operations which are allowed to enable repair based node operations. The operations can be bootstrap, replace, removenode, decommission and rebuild.")
     , enable_compacting_data_for_streaming_and_repair(this, "enable_compacting_data_for_streaming_and_repair", liveness::LiveUpdate, value_status::Used, true, "Enable the compacting reader, which compacts the data for streaming and repair (load'n'stream included) before sending it to, or synchronizing it with peers. Can reduce the amount of data to be processed by removing dead data, but adds CPU overhead.")
+    , repair_partition_estimation_ratio(this, "repair_partition_estimation_ratio", liveness::LiveUpdate, value_status::Used, 0.1,
```

Comment:
we should have the word `count` in the name, becuase currently it is unclear what quality of a partiiton we estimate here. I suggest: `repair_partition_count_estimate_fraction`.

Reply:
Originally, I had a word number in the name which I dropped. The count looks good. Changed in v2.